pyannote/audio/interactive/common/utils.py

```diff
@@ -155,4 +161,9 @@ def before_db(examples):
                 for span in eg[key]
             ]
 
+        # 3. Change label span name (for diarization recipe)
+        for span in eg["audio_spans"]:
+            if span["label"] in eg:
+                span["label"] = eg[span["label"]]
+
```

Comment:
What does this do exactly? 

If it is specific to a particular recipe, I'd rather have the latter use its own `before_db`.